tests/src/test/scala/cats/tests/ApplicativeErrorTests.scala

```diff
@@ -0,0 +1,42 @@
+package cats
+package tests
+
+import cats.data.{Xor, XorT}
+
+class ApplicativeErrorCheck extends CatsSuite {
+
+  type ErrorOr[A] = String Xor A
+
+  val failed: String Xor Int =
+    "Badness".raiseError[ErrorOr, Int]
+
+  test("raiseError syntax creates an Xor with the correct type parameters") {
+    failed should === ("Badness".left[Int])
+  }
+
+  test("handleError syntax transforms an error to a success") {
+    failed.handleError(error => error.length) should === (7.right)
+  }
+
+  test("handleErrorWith transforms an error to a success") {
+    failed.handleErrorWith(error => error.length.right) should === (7.right)
+  }
+
+  test("attempt syntax creates a wrapped Xor") {
+    failed.attempt should === ("Badness".left.right)
+  }
+
+  test("attemptT syntax creates an XorT") {
+    type ErrorOrT[A] = XorT[ErrorOr, String, A]
```

Comment:
This doesn't appear to be used. Please tell me that it's mere presence doesn't affect whether the next line compiles? :|